wremnants/muon_efficiencies_smooth.py

```diff
@@ -345,7 +347,7 @@ def make_muon_efficiency_helpers_smooth(filename = data_dir + "/muonSF/allSmooth
 
                     # temporary patch for some missing SF histograms in 2D (for the 3D case we have everything)
                     # only relevant for dilepton (no warning needed otherwise)
-                    if isoDefinition != "iso04vtxAgn":
+                    if isoDefinition != "iso04vtxAgn" or era=="2018":
```

Comment:
same here

Reply:
But this we checked no, if we don't give era==2018 the code crashes, due to antitrigger SFs